File: Capítulo_8/Dicionários/Métodos clear, size, isEmpty e toString/clear.js
import { defaultToString } from "../../ultil/util.js";
export default class Dictionary {
    constructor(toStrFn = defaultToString) {
        this.toStrFn = toStrFn;
        this.table = {};
    }

    // clear(): esse método remove todos os valores do dicionário.
    clear() {
        this.table = {};
    }
}

File: Capítulo_8/Dicionários/Métodos clear, size, isEmpty e toString/Códigos em Python/clear.py
# defaultToString:
def defaultToString(item):
    if item is None:
        return "NONE"
    elif isinstance(item, str):
        return f"{item}"
    return str(item)

# Classe Value Pair
class ValuePair:
    def __init__(self, key, value):
        self.key = key
        self.value = value
    
    def __str__(self):
        return f"[#{self.key}: {self.value}]"

# Classe Dictionary
class Dictionary:
    def __init__(self, to_str_fn = defaultToString):
        self.to_str_fn = to_str_fn
        self.table = {}

    # clear(): esse método remove todos os valores do dicionário.
    def clear(self):
        self.table = {}
